Update AutoRest.CSharp Version

ellismg opened the pull request:
The automation is still broken, I opened https://github.com/Azure/autorest.csharp/issues/1506 in the Autorest.CSharp repository to track a fix.  But for now, manually updating the version which pulls in some code generator changes for LLC (the changes here are due to a push to share more logic between the the Track 2 Data Plane generator and the LLC generator).

ellismg committed:
Update AutoRest.CSharp Version

christothes commented on sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerIdentityServiceClient.cs:
```diff
@@ -17,7 +17,8 @@ namespace Azure.Security.ConfidentialLedger
     {
         /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
-        public virtual HttpPipeline Pipeline { get; }
+        public virtual HttpPipeline Pipeline { get => _pipeline; }
```
This requires a change to https://github.com/Azure/azure-sdk-for-net/blob/13e36f97bb2aa23b9c9c60cea890dbc18a1eb5b5/sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerIdentityServiceClient.cs#L35

ellismg replied:
Thanks, Chris.  There were a handful of the same issue across some other libraries as well, which I went ahead and fixed up.

ellismg committed:
Use `_pipeline` instead of `Pipeline`

The codegen has changed slightly for LLCs to bring it in line with
other Track 2 clients. Hand authored code that was explicitly setting
`Pipeline` should now instead set `_pipeline` as `Pipeline` is now a
read only property.

ellismg committed:
Supress unneeded members in WebPubSub
